.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Oh no, a bug. It happens. Thanks for reporting an issue with NewPipe
+
+Use this template to notify us if you found a bug.
+
+To make it easier for us to help you please enter detailed information below.
+
+Please note, we only support the latest version of NewPipe and
```

Comment:
You didn't include a link to the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/wiki/CONTRIBUTING.md-mirror)

Also, question to TeamNewpipe
> Do not report crashes in the GitHub issue tracker. NewPipe has an automated crash report system that will ask you to send a report via e-mail when a crash occurs. This contains all the data we need for debugging, and allows you to even add a comment to it. You'll see exactly what is sent, the system is 100% transparent

You don't want people to just paste the crash report on GitHub because Sentry is better with automatization, but you are welcoming detailed report with steps to reproduce, further explaination… on GitHub, am I right?
Because if yes, this paragraph is not needed, by following the template people will give information. If no, we should write about that.

Reply:
>You didn't include a link to the contribution guidelines

Not needed for a bug/feature report, not like they are actually contributing.

Reply:
> Not needed for a bug/feature report, not like they are actually contributing.

The guidelines contain valuable information about reporting bugs or requesting features. Also, a simple link doesn't force people to follow it (although it would be better).

> You don't want people to just paste the crash report on GitHub because Sentry is better with automatization, but you are welcoming detailed report with steps to reproduce, further explaination… on GitHub, am I right?

I think we need to make a poll with the regular contributors asking them about their usage of Sentry. To me it doesn't seem overly well used.

Depending on the result we have to define how bugs should be reported. Perhaps we can include a unique (enough) ID or something so that issues sent to Sentry can be found more easily on demand (i.e., the reporter can post that ID, and the devs can find the relevant Sentry entry by searching for it).

*Not* posting all the bug report information in public is also a good idea for privacy reasons. Why should we request anyone to post intimate details about their YouTube habits for bug reports?

We also intend to switch to an alternative/fork/successor, as Sentry has turned into a proprietary software lately*.

In any case, @TobiGr, it's up to you to decide how bugs should be reported. I just provided some ideas for improvements.

*(*Damn all those generous CLAs from companies! They really make sense for non-profit projects only if they're restricted to stuff relevant for copyright cases in courts (i.e., they need to be worded in a way so the license can't be changed but the company has the copyright "officially" so they can actually sue others), and non-profits don't misuse them for turning software proprietary...)*

Reply:
Good point with the link, will do some magic formatting and include it

Reply:
Done, will leave it open for Tobis comment

Reply:
> Perhaps we can include a unique (enough) ID or something so that issues sent to Sentry can be found more easily on demand (i.e., the reporter can post that ID, and the devs can find the relevant Sentry entry by searching for it).

Good idea. In this way, we could also add more valuable information like downloaded documents to the crash report.

> I think we need to make a poll with the regular contributors asking them about their usage of Sentry. To me it doesn't seem overly well used.

The problem is that Sentry is too messy and searching logs in Sentry is quite difficult. We might also consider to not accept bugs from older versions.